packages/wallet-sdk/package.json

```diff
@@ -28,14 +28,18 @@
     "lint": "eslint . --ext .ts,.tsx --fix"
   },
   "dependencies": {
+    "@noble/ciphers": "^0.5.3",
+    "@noble/curves": "^1.4.2",
     "@noble/hashes": "^1.4.0",
     "clsx": "^1.2.1",
     "eventemitter3": "^5.0.1",
+    "expo-standard-web-crypto": "^1.8.1",
```

Comment:
this polyfills the `crypto.getRandomValues`, which is needed in "@noble/curves" to provide randomness

Reply:
this could be moved to optional dependencies instead since nothing besides React Native needs it

Reply:
or ideally polyfilling Web Crypto should be done not by a library but a React Native application. It's not a library's job to polyfill functionality imo

Reply:
thx @talentlessguy, agreed and we will omit the polyfill here! 

also would like to know your view on the approach we took on RN Cipher if possible. Generally we will ask consumer to provide randomness and use @noble/curves to achieve diffie hellman, do you have any concern or maybe you know any potential hiccup in this approach?

Reply:
perhaps there's no guarantee that provided randomness is random enough and isn't using something like `Math.random()`